src/components/CompletedTask/CompletedTask.tsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from "react";
+
+import "./CompletedTask.scss";
+import {
+  Card,
+  CardContent,
+  Typography,
+  CardHeader,
+  CardMedia,
+  Collapse,
+  IconButton,
+  IconButtonProps,
+  styled,
+} from "@mui/material";
+
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+interface ExpandMoreProps extends IconButtonProps {
+  expand: boolean;
+}
+
+const ExpandMore = styled((props: ExpandMoreProps) => {
```

Comment:
Can you style this without the use of styled components?

Reply:
We've removed the styled element